Update README.md to reflect 7.4 EAR (#708)
rticommunity/rticonnextdds-examples: README.md

The examples contained in the
[master](https://github.com/rticommunity/rticonnextdds-examples/tree/master)

> branch of this repository have been built and tested against **RTI Connext 7.3.0 LTS**.

Reviewer: ```suggestion
branch of this repository were built and tested against **RTI Connext 7.3.0 LTS**.
```

If you are using a newer feature release, check out the appropriate branch:

- [release/7.4.0](https://github.com/rticommunity/rticonnextdds-examples/tree/release/7.4.0)

>   (early-access release)

Reviewer: ```suggestion
  (early access release)
```
